routers/web/repo/actions/view.go

```diff
@@ -0,0 +1,306 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package actions
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	actions_model "code.gitea.io/gitea/models/actions"
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/models/unit"
+	"code.gitea.io/gitea/modules/actions"
+	context_module "code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/timeutil"
+	"code.gitea.io/gitea/modules/util"
+	"code.gitea.io/gitea/modules/web"
+	actions_service "code.gitea.io/gitea/services/actions"
+
+	runnerv1 "code.gitea.io/actions-proto-go/runner/v1"
+	"xorm.io/builder"
+)
+
+func View(ctx *context_module.Context) {
```

Comment:
That's the function for `http://<instance>/test/test/actions/runs/1/jobs/0`, right?
This function doesn't load at all for me at the moment.
It simply loads for an eternity without showing why.
I'll try debugging _why_.

Reply:
Hmm, interesting.
I restarted the instance without changes except adjusting the log level, and now it works without problems?

Reply:
Hmm, even more interesting.
Now it appears again.
And the logs don't even say anything apart from that apparently the request took roughly `3.5s`.
Funny thing: This response doesn't seem to reach the frontend, that is forever loading.